### PR TITLE
Adding the developer mode requirement to PWA widgets doc

### PR DESCRIPTION
**Pull request by captainbrosset**
The Developer Mode is needed on Windows 11 to install PWA widgets, for now. Somehow, we missed this from the docs. So adding it now, in this PR.

Preview:

* Preview server: https://review.learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/widgets?branch=pr-en-us-2800#install-winappsdk-and-enable-developer-mode
* GitHub: https://github.com/MicrosoftDocs/edge-developer/blob/user/pabrosse/widget-dev-mode/microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md#install-winappsdk-and-enable-developer-mode

AB#46125268

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md [hunk 1]
```diff
@@ -35,9 +35,12 @@ See also:
...
 
 <!-- ====================================================================== -->
-## Install WinAppSDK
+## Install WinAppSDK and enable Developer Mode
 
-To enable developing widgets on your local machine, install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
+To enable developing and testing widgets on your local machine:
+
+* Install [WinAppSDK 1.2](/windows/apps/windows-app-sdk/older-downloads#windows-app-sdk-12).
+* Enable Developer Mode in Windows 11: go to **Settings** > **For developers** and then enable **Developer Mode**.
```
Comment (amandabaker) [inline microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:43]: Should this be 
```suggestion
* Enable Developer Mode in Windows 11: go to **Settings** > **Privacy & security** >  **For developers** and then enable **Developer Mode**.
```

**Review by amandabaker — COMMENTED** [review]
LGTM aside from the one comment

File: microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md (hunk 1 above)
Comment (captainbrosset) [reply microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:43]: Oddly, I don't have the option under **Settings** > **Privacy & security**. I have it under **Settings** > **System**. And I also have it on the **Settings** landing page (but maybe that default list is built from settings I've accessed recently).

Comment (captainbrosset) [reply microsoft-edge/progressive-web-apps-chromium/how-to/widgets.md:43]: Nevermind, the **Settings** app opens up _on_ the **System** tab. And that's where I see **For developers**.

**Review by amandabaker — APPROVED** [review]

**Review by amandabaker — APPROVED** [review]
LGTM! Thanks for getting this updated!

**Review by mikehoffms — APPROVED** [review]

**Pull request merged**